### DISCUSSION TITLE
Updating Credentials

### DISCUSSION THREAD

**Discussion by TScalzott** [dahlb/ha_carrier#83] [Q&A] [answered]
Maybe I am missing something simple, but I cannot determine how to update my credentials used by this integration after updating them with Carrier.

Configuring the integration via the UI only lets me change the polling interval and otherwise loading the integration is failing due to the changed credentials.

I don't wish to uninstall and re-install, since I assume that will remove entities that are in use and impact my dashboards and automations.

**Comment by dahlb** [comment, marked as answer]
Just re-add as discussed [here](https://github.com/dahlb/ha_hatch/issues/104), the integration is keyed to the username so it should be seamlessly updated

**Reply by TScalzott** [reply]
I don't see an "Add Entry", but adding a hub let me update the credentials.
